### PR TITLE
Daniel benson

### PR DESCRIPTION
**Pull request by Daniel-Benson-Poe**
# Description
Updated csv file and jsonified it for api endpoint

# Number of Files changed
7

# Questions for Reviewer

# Type of Change
- [ ] New Feature
- [x ] Feature Update
- [ ] Fix Error

# Status
- [ x] Tested

### CODE REVIEW

File: project/app/api/update.py [hunk 1]
```diff
@@ -216,11 +216,11 @@ def listify(text):
     df = df[[
         'id', 'state', 'city',
         'date', 'title', 'description',
-        'links', 'geocoding'
+        'links', 'lat', 'long'
```
Comment (KristineYW) [inline project/app/api/update.py:219]: We're using 'src' instead of 'links' for the source urls

File: project/app/api/update.py [hunk 2]
```diff
@@ -216,11 +216,11 @@ def listify(text):
     df = df[[
         'id', 'state', 'city',
         'date', 'title', 'description',
```
Comment (KristineYW) [inline project/app/api/update.py:218]: We're using 'desc' instead of 'description' for column name to send to BE

File: project/app/main.py [hunk 3]
```diff
@@ -77,11 +79,12 @@ def run_update() -> None:
...
     Update backlog database with data from reddit.
     '''
-    print('Updating backlog at %s' % datetime.now())
 
+
+    # globalize these variables because I need to
     PRAW_CLIENT_ID = os.getenv('PRAW_CLIENT_ID')
     PRAW_CLIENT_SECRET = os.getenv('PRAW_CLIENT_SECRET')
-    PRAW_USER_AGENT = os.getenv('PRAW_USER_AGENT')
+    PRAW_USER_AGENT =  os.getenv('PRAW_USER_AGENT')
```
Comment (KristineYW) [inline project/app/main.py:87]: Unnecessary space added here (probably accidentally)

File: project/app/main.py [hunk 4]
```diff
@@ -238,12 +233,15 @@ def listify(text):
     df = df[[
         'id', 'state', 'city',
         'date', 'title', 'description',
-        'links', 'geocoding'
+        'links', 'lat', 'long'
```
Comment (KristineYW) [inline project/app/main.py:236]: 'description' and 'links' should be 'desc' and url' unless these are changed in the JSONified file by backend or elsewhere

**Review by KristineYW — CHANGES_REQUESTED** [review]
See comments. Looks great! There were 7 files changed, not 2.

**Review by KristineYW — APPROVED** [review]
Fixes look good!!

**Review by Griffinw15 — APPROVED** [review]
looks good

**Pull request merged**